src/buildblock/ProjData.cxx

```diff
@@ -225,6 +229,27 @@ read_from_file(const string& filename,
 //  this->exam_info_sptr.reset(new ExamInfo(new_exam_info));
 //}
 
+
+ProjDataInMemory
+ProjData::get_subset(const std::vector<int>& views) const
+{
+  auto subset_proj_data_info_sptr =
+    std::make_shared<ProjDataInfoSubsetByView>(proj_data_info_sptr, views));
+  ProjDataInMemory subset_proj_data(exam_info_sptr, subset_proj_data_info_sptr);
+
+  for (int segment_num=get_min_segment_num(); segment_num<=get_max_segment_num(); ++segment_num)
+    {
+      for (int subset_view_num=0; subset_view_num<views.size(); ++subset_view_num)
+        {
+          const Viewgram<float> viewgram = this->get_viewgram(views[subset_view_num], segment_num);
+          Viewgram<float> subset_viewgram = subset_proj_data_info_sptr-> get_empty_viewgram(subset_view_num, segment_num);
+          subset_viewgram.fill(viewgram);
+          if (subset_proj_data.set_viewgram(subset_viewgram) != Succeeded::yes)
+            error("ProjData::get_subset failed to set a viewgram");
+        }
+    }
+}
```

Comment:
`return subset_proj_data` missing?

Reply:
yes indeed. feel free to commit (with `[ci skip]`)

Reply:
@KrisThielemans committed but cannot push to `ashgillman/STIR`

Reply:
ah, @ashgillman maybe you fix this? For now, @evgueni-ovtchinnikov  just do a PR on @ashgillman's branch.

Reply:
I've committed this now. thanks for spotting.